backend/app/controllers/authentication_controller.py

```
@@ -21,15 +29,18 @@
 
 SECRET_KEY = os.getenv("SECRET_KEY")
 ALGORITHM = "HS256"
-ACCESS_TOKEN_EXPIRE_MINUTES = 60
+ACCESS_TOKEN_EXPIRE_MINUTES = 20  # 30 seconds
```

Flake8 (7.2.0)

[error] redefinition of unused 'ACCESS_TOKEN_EXPIRE_MINUTES' from line 10

(F811)